lib/web_ui/lib/platform_isolate.dart

```diff
@@ -0,0 +1,32 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+part of ui;
+
+/// Runs [computation] on the platform thread and returns the result.
+///
+/// Internally this creates an isolate on the platform thread that will be
```

Comment:
This here is web, so it runs it immediately.

If you want to have the same comment on web and non-web version, then please weaken the promises, say: "it may run the computation on another isolate and as part of that copy the closure ..."